ledger/narwhal/subdag/src/bytes.rs

```diff
@@ -21,18 +21,21 @@ impl<N: Network> FromBytes for Subdag<N> {
         let version = u8::read_le(&mut reader)?;
         // Ensure the version is valid.
         // TODO (howardwu): For mainnet - Change the version back to 1.
-        if version != 1 && version != 2 {
+        if version != 1 && version != 2 && version != 3 {
```

Comment:
```suggestion
        if !(1..=3).contains(&version) {
```

Reply:
Noting the comment just above it, I think it will be easier to revert if we have my ugly version checks.